vars/cloud.groovy

```diff
@@ -182,8 +182,8 @@ void dockerBuildMultiPlatformImages(String buildImageTag, List platforms, boolea
 *
 * You should have run `prepareForDockerMultiplatformBuild` method before executing this method
 */
-void dockerBuildPlatformImage(String buildImageTag, String platform) {
-    sh "docker buildx build --push --sbom=false --provenance=false --platform ${platform} -t ${buildImageTag} ."
+void dockerBuildPlatformImage(String buildImageTag, String platform, boolean outputToFile = false) {
+    sh "docker buildx build --push --sbom=false --provenance=false --platform ${platform} -t ${buildImageTag} .${outputToFile ? ' > ' + buildImageTag + '-' + platform + '-build.log' : ''}"
```

Comment:
```suggestion
    def logFileName = (buildImageTag + '-' + platform + '-build.log')
        .replaceAll('/','_')
        .replaceAll(':','_')
    sh "docker buildx build --push --sbom=false --provenance=false --platform ${platform} -t ${buildImageTag} .${outputToFile ? ' 2> ' + "${WORKSPACE}/${logFileName}" : ''}"
```

Reply:
Solving issues:
* `/` in path which would result in non-existent parent dir error.
* putting the log file into workspace, that is mapped from the host machine and does survive until post>always phase where we store artifacts.